pyproject.toml

```diff
@@ -1,4 +1,6 @@
 [tool.pylint.master]
+exit-zero = true
```

Comment:
Wait a second: you should not always exit with zero!

The first check on errors should actually crash if failing.